modules/ppcp-settings/resources/js/Components/Screens/Overview/TabSettingsElements/Blocks/LiveAccount.js

```diff
@@ -0,0 +1,206 @@
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import {
+	AccordionSettingsBlock,
+	ButtonSettingsBlock,
+	RadioSettingsBlock,
+	ToggleSettingsBlock,
+	InputSettingsBlock,
+} from '../../../../ReusableComponents/SettingsBlocks';
+import TitleBadge, {
+	TITLE_BADGE_POSITIVE,
+} from '../../../../ReusableComponents/TitleBadge';
+import ConnectionInfo, {
+	connectionStatusDataDefault,
+} from '../../../../ReusableComponents/ConnectionInfo';
+
+const LiveAccount = ( { settings, updateFormValue } ) => {
+	const className = settings.sandboxConnected
+		? 'ppcp-r-settings-block--sandbox-connected'
+		: 'ppcp-r-settings-block--sandbox-disconnected';
+
+	return (
+		<AccordionSettingsBlock
+			title={ __( 'Live Payments', 'woocommerce-paypal-payments' ) }
+			className={ className }
+			description={
+				settings.sandboxConnected
```

Comment:
`sandboxConnected` looks wrong. Is this really the correct prop? If yes, then we should rename it to `isConnected`